tests/test_model.py

```diff
@@ -0,0 +1,27 @@
+import asynctest
+import asyncio
+import pytest
+from lsst.ts import adamSensors
+
+
+class ModelTestCase(asynctest.TestCase):
+    async def test_connect(self):
+        m = adamSensors.AdamModel(simulation_mode=True)
+        await m.connect("fakeIP", 502)
+        assert m is not None
+
+    async def test_read_voltage(self):
+        m = adamSensors.AdamModel(simulation_mode=True)
+        await m.connect("fakeIP", 502)
+        v1 = await m.read_voltage()
+
+        # check the min, max, and zero-ish values from simulator
+        assert v1[1] == pytest.approx(-10)
+        assert v1[2] == pytest.approx(10)
+        assert v1[3] == pytest.approx(0, abs=1e-3)
+        assert v1[4] == pytest.approx(0, abs=1e-3)
+
+        # check that channel 0 is changing over time
+        await asyncio.sleep(2)
+        v2 = await m.read_voltage()
+        assert v1[0] != v2[0]
```

Comment:
I don;t think this unit test will run when you do pytest. you need to add this at the bottom:

```
if __name__ == "__main__":
    unittest.main()
```